test/it/parser/src/main/java/org/apache/shardingsphere/test/it/sql/parser/external/loader/ExternalSQLParserTestParameterLoader.java

```diff
@@ -17,69 +17,35 @@
 
 package org.apache.shardingsphere.test.it.sql.parser.external.loader;
 
-import lombok.RequiredArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
 import org.apache.shardingsphere.test.it.sql.parser.external.ExternalSQLParserTestParameter;
-import org.apache.shardingsphere.test.it.sql.parser.external.loader.strategy.TestParameterLoadStrategy;
-import org.apache.shardingsphere.test.it.sql.parser.external.loader.summary.FileSummary;
+import org.apache.shardingsphere.test.loader.AbstractTestParameterLoader;
+import org.apache.shardingsphere.test.loader.strategy.TestParameterLoadStrategy;
 
-import java.io.BufferedReader;
-import java.io.IOException;
-import java.io.InputStreamReader;
-import java.net.URI;
 import java.util.Collection;
 import java.util.LinkedList;
-import java.util.Map;
-import java.util.Map.Entry;
-import java.util.stream.Collectors;
 
 /**
  * External SQL parser test parameter loader.
  */
-@RequiredArgsConstructor
 @Slf4j
-public final class ExternalSQLParserTestParameterLoader {
+public final class ExternalSQLParserTestParameterLoader extends AbstractTestParameterLoader<ExternalSQLParserTestParameter> {
     
-    private final TestParameterLoadStrategy loadStrategy;
+    public ExternalSQLParserTestParameterLoader(final TestParameterLoadStrategy loadStrategy) {
+        super(loadStrategy);
+    }
     
     /**
-     * Load SQL parser test parameters.
-     *
-     * @param sqlCaseURI SQL case URI
-     * @param resultURI result URI
+     * create external sql parser test parameters.
```

Comment:
1. Please keep blank line between javadoc and @param
2. `sql` should be `SQL` in javadoc